ghcide/session-loader/Development/IDE/Session.hs

```diff
@@ -758,7 +761,7 @@ getDependencyInfo fs = Map.fromList <$> mapM do_one fs
 -- ID. Therefore we create a fake one and give them all the same unit id.
 removeInplacePackages
     :: UnitId     -- ^ fake uid to use for our internal component
-    -> [UnitId]
+    -> NonEmpty UnitId
```

Comment:
Perhaps simpler to just convert to a normal list before calling this - I don't think _this_ function needs to require its argument to be non-empty!